### PR TITLE
(DOCSP-1179): Added release notice for v2.3 beta.

### PR DESCRIPTION
**Pull request by atsansone**
@rkumar-mongo : Simple add of the v2.3 changes.

### CODE REVIEW

File: source/includes/release-notes/fact-release-notes-2-3.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
+|bi| version 2.3 improves enterprise deployment support. The new
+features include:
+
+Improved Schema Management
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+|bi-short| has been simplified to a single component.
+The schema sampling and mapping features of
+:doc:`mongodrdl </reference/mongodrdl>` have added to
```
Comment (rkumar-mongo) [inline source/includes/release-notes/fact-release-notes-2-3.rst]: "have added to" reads a little odd. Maybe just 'are included in'

Comment (rkumar-mongo) [reply source/includes/release-notes/fact-release-notes-2-3.rst]: Alternatively, just say ":doc:`mongosqld </reference/mongosqld>` now includes the schema mapping functionality previously in :doc:`mongodrl`. `mongodrdl` is no longer necessary in |bi-short| 2.3.

File: source/includes/release-notes/fact-release-notes-2-3.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+:doc:`mongosqld </reference/mongosqld>`.
+The :doc:`mongosqld </reference/mongosqld>` maps schemas itself.
+
+Kerberos Authentication Support
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+|bi-short| can now use `Kerberos <https://web.mit.edu/kerberos/>`_,
+an enterprise security protocol that authenticates clients through
+a trusted third server.
```
Comment (rkumar-mongo) [inline source/includes/release-notes/fact-release-notes-2-3.rst]: Do we have docs on this yet? If so I'd link to that here. If not then we should follow up at some point to make sure that gets linked out.

File: source/includes/release-notes/fact-release-notes-2-3.rst [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+Log Rotation
+~~~~~~~~~~~~
+
+With version 2.3, :doc:`mongosqld </reference/mongosqld>` can rotate its
+logs upon command. When the log rotate command is issued, the existing
```
Comment (rkumar-mongo) [inline source/includes/release-notes/fact-release-notes-2-3.rst]: What command? like a system kill? SIGTERM? If there is a specific page where we detail how to do it, link there.

Comment (rkumar-mongo) [inline source/includes/release-notes/fact-release-notes-2-3.rst]: "When mongosqld receives a log rotate command, the existing log file is closed and a new empty log file is opened."

Also, naming conventions? If there's not a dedicated page we should talk about it somewhere.

File: source/index.txt [hunk 4]
```diff
@@ -28,20 +28,24 @@ presents a challenge when working with MongoDB's
 :manual:`flexible schema </data-modeling>` and rich,
 multi-dimensional documents.
 
-The |bi| acts as a layer that passes queries and
-data between a :program:`mongod` or :program:`mongos` instance and your
-reporting tool. The |bi| stores no data, and purely serves to bridge your
-MongoDB server with business intelligence tools.
+The |bi| acts as a layer that passes queries and data between a
+:program:`mongod` or :program:`mongos` instance and your reporting tool.
+The |bi| stores no data, and purely serves to bridge your MongoDB server
```
Comment (rkumar-mongo) [inline source/index.txt]: /s/server/cluster ? A server implies a single server, e.g. a standalone. Moreso given a sharded cluster.

File: source/includes/release-notes/fact-release-notes-2-3.rst (hunk 3 above)
Comment (atsansone) [reply source/includes/release-notes/fact-release-notes-2-3.rst]: There is a page. I included it.

File: source/index.txt (hunk 4 above)
Comment (atsansone) [reply source/index.txt]: Fixed.

Comment (rkumar-mongo) [reply source/index.txt]: I think the change didn't get pushed?

**Comment by breinero** [comment]
Log Rotation marked as "(beta)" seems redundant, but other than that it LGTM

**Comment by atsansone** [comment]
Merged 5e122883caefdb0b1e95ccc4a6fb27c6a51e8925

**Pull request closed without merging**